### PR TITLE
Awesome books

### PR DESCRIPTION
**Pull request by oluwajuwon8**
This an awesome book project.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+const displayBooks = () => {
+  const booksData = localStorage.getItem('booksData');
+  const convertedBooks = JSON.parse(booksData);
+  document.getElementById('bookstatus').innerHTML = 'No books added';
+  if (convertedBooks && convertedBooks.length === 0) {
+    document.getElementById('bookstatus').innerHTML = 'No books added';
+  } else {
+    document.getElementById('bookstatus').innerHTML = '';
+    let html = '';
+    convertedBooks.forEach((book) => {
```
Comment (henatan99) [inline script.js:37]: - [ ] During the initial load of the page, the `convertedBooks` data does not exist in the local storage. As a result the `forEach` method would throw an error as `convertedBooks` is evaluated as `null`. Kindly fix this issue. :pray: 

Screenshot from 2021-11-30 13-45-37

File: script.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  } else {
+    Books.push(book);
+    const convertedBooks = JSON.stringify(Books);
+    localStorage.setItem('booksData', convertedBooks);
+    window.location.reload();
+  }
+};
+
+// eslint-disable-next-line no-unused-vars
+function removeBook(id) {
```
Comment (henatan99) [inline script.js]: - [ ] To follow Javascript best practices, kindly use the arrow function. :pray:

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @oluwajuwon8 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ArthurGC — APPROVED** [review]
# Approved project!  💯 ✅ 🥳 🎈 🎉
Hi  @chukwuemeka1234 @oluwajuwon8 

Amazing job implementing suggestions from the previous reviewer! 💯 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**